lib/charms/operator_libs_linux/v0/juju_systemd_notices.py

```diff
@@ -111,12 +113,11 @@ def _on_slurmd_stopped(self, _: ServiceStoppedEvent) -> None:
 
 # juju-systemd-notices charm library dependencies.
 # Charm library dependencies are installed when the consuming charm is packed.
-PYDEPS = ["dbus-fast>=1.90.2"]
+PYDEPS = ["dbus-fast>=1.90.2", "pyyaml>=6.0.1"]
```

Comment:
Couple of minor things here: `PyYAML` seems to be the canonical spelling/case of the [project name on PyPI](https://pypi.org/project/PyYAML/). Would `PyYAML==6.*` be better, so that it doesn't pull in a potentially-breaking 7.x version?

Reply:
Also, we should definitely update LIBPATCH. Ideally we should do this change without having to update LIBAPI (that is, no breaking changes).

Reply:
Dropped `PyYAML` as a dependency in https://github.com/canonical/operator-libs-linux/pull/128/commits/c8f0767563b3e317b91c0406c0872db930a09830